Source/Core/DolphinWX/AboutDolphin.cpp

```diff
@@ -17,19 +17,47 @@
 #include <wx/windowid.h>
 #include <wx/generic/statbmpg.h>
 
+#ifdef __APPLE__
+#import <AppKit/AppKit.h>
+#endif
+
 #include "Common/Common.h"
 #include "DolphinWX/AboutDolphin.h"
 #include "DolphinWX/resources/dolphin_logo.cpp"
 
+static void BanishBackground(wxTextCtrl* ctrl)
+{
+#ifdef __APPLE__
+       NSTextField* tf = (NSTextField*)ctrl->GetHandle();
+       tf.drawsBackground = NO;
+#endif
```

Comment:
The idea is not to set the background color, but to remove it entirely. Setting it to wxTransparentColour didn't remove it; it's still white.